tests/app/questionnaire/test_value_source_resolver.py

```diff
@@ -1165,53 +1180,52 @@ def test_supplementary_data_value_source_list_items(
     )
 
 
-def test_supplementary_data_invalid_selector_raises_exception(
-    supplementary_data_store_with_data,
+def test_supplementary_data_value_source_list_items_extra_item_not_resolving_when_no_value(
```

Comment:
I find this hard to understand. What is it testing?. `extra item not resolving when no value`?. It's not clear what that means.

Reply:
Added the initial description you used when requesting this test.